packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -254,8 +254,36 @@ class WebViewController {
     return _channel.invokeMethod("reload");
   }
 
-  Future<void> _updateSettings(Map<String, dynamic> update) async {
-    return _channel.invokeMethod('updateSettings', update);
+  Future<void> _updateSettings(_WebSettings update) async {
```

Comment:
nit: s/update/newSettings/ (this is no longer the delta but the complete new settings)